02-express-tutorial/controllers/services.js

```diff
@@ -0,0 +1,49 @@
+// Note: This could be moved to a separate folder as /services
```

Comment:
In this class we are organizing our code by its function (e.g. "controllers", "models", etc).
This file doesn't really contain controller logic.  It would be better to move this to another folder e.g. "services"